kghDataManagement/templates/kghDataManagement/kgh_upload_page.html

```diff
@@ -0,0 +1,68 @@
+{% extends "itemManagement/chrome_template.html" %}
+{% load static %}
+{% block body %}
+
+    <div class="jumbotron mx-auto col-lg-8">
+        <h1 class="mx-auto text-center">Upload Your KGH Catalog</h1>
+
+        <br>
+        <form method="POST" enctype="multipart/form-data">
+            <div class="form-group">
+                {% csrf_token %}
+
+                <label for="uploadKghFile">Upload the new KGH catalog here to update item prices:</label>
+                {#                TODO, more file types#}
+                <div class="row">
+                    <input type="file" class="form-control-file col-6" accept=".csv" id="uploadKghFile" name="kghFile">
```

Comment:
Should also allow the csv mimetype "text/csv"

Reply:
Will do. What's the difference?

Reply:
It allows the browser to accept CSV files that don't have an extension (the norm for Linux)